### PR TITLE
[Operator] Add gelu backward

### PR DESCRIPTION
**Pull request by BruceDai003**

### CODE REVIEW

File: src/flag_gems/ops/gelu.py [hunk 1]
```diff
@@ -31,9 +32,56 @@ def gelu_tanh(x):
     return output
 
 
+@pointwise_dynamic(promotion_methods=[(0, 1, "DEFAULT")])
+@triton.jit
+def gelu_backward_none(x, dy):
+    scale1: tl.constexpr = 0.7071067811
```
Comment (iclementine) [inline src/flag_gems/ops/gelu.py]: Add some comment on the value of those constants.

```python
scale1: tl.constexpr = 1 / math.sqrt(2)
scale2: tl.constexpr = 1 / math.sqrt(2 * math.pi)
``` 

to help understanding the formula.

File: src/flag_gems/ops/gelu.py [hunk 2]
```diff
@@ -31,9 +32,56 @@ def gelu_tanh(x):
...
+    )
+    dx = dydx * dy
+    return dx
+
+
+@pointwise_dynamic(promotion_methods=[(0, 1, "DEFAULT")])
+@triton.jit
+def gelu_backward_tanh(x, dy):
+    x_fp32 = x.to(tl.float32)
+    tanh_out = tanh(0.79788456 * x * (1 + 0.044715 * pow(x_fp32, 2)))
```
Comment (iclementine) [inline src/flag_gems/ops/gelu.py:55]: Add some comment on the constant

`0.79788456 # math.sqrt(2 / math.pi)`

**Review by iclementine — DISMISSED** [review]
LGTM

File: src/flag_gems/ops/gelu.py (hunk 2 above)
Comment (BruceDai003) [reply src/flag_gems/ops/gelu.py:55]: ok. I used sympy package to help derive the derivatives, so for the other constant "0.1070322243", I actually didn't try to understand where it comes from. :(

**Review by iclementine — APPROVED** [review]

**Pull request merged**